patches/minecraft/net/minecraft/entity/Entity.java.patch

```diff
@@ -46,7 +46,23 @@
      public int func_145782_y()
      {
          return this.field_145783_c;
-@@ -1053,7 +1065,7 @@
+@@ -344,6 +356,7 @@
+         this.field_70165_t = p_70107_1_;
+         this.field_70163_u = p_70107_3_;
+         this.field_70161_v = p_70107_5_;
++        if (this.isAddedToWorld) this.field_70170_p.func_72866_a(this, false); // Forge - Process chunk registration after moving.
+         float f = this.field_70130_N / 2.0F;
+         float f1 = this.field_70131_O;
+         this.func_174826_a(new AxisAlignedBB(p_70107_1_ - (double)f, p_70107_3_, p_70107_5_ - (double)f, p_70107_1_ + (double)f, p_70107_3_ + (double)f1, p_70107_5_ + (double)f));
+@@ -995,6 +1008,7 @@
+         this.field_70165_t = (axisalignedbb.field_72340_a + axisalignedbb.field_72336_d) / 2.0D;
+         this.field_70163_u = axisalignedbb.field_72338_b;
+         this.field_70161_v = (axisalignedbb.field_72339_c + axisalignedbb.field_72334_f) / 2.0D;
++        if (this.isAddedToWorld) this.field_70170_p.func_72866_a(this, false); // Forge - Process chunk registration after moving.
```

Comment:
Will this cause loading of a ton of chunks if the entity moves too fast (e.g. a minecart)?